BoxContentSDK/BoxContentSDK/OAuth2/BOXAuthorizationViewController.m

```diff
@@ -274,7 +250,7 @@ - (void)failAuthenticationWithConnection:(NSURLConnection *)connection
             self.completionBlock(self, nil, self.connectionError);
         }
     }
-    
+
```

Comment:
We should discuss offline, as I'm not sure exactly what you mean. The completion block here is only called at the last step as-is, on handling the failure... That in turn either gets called from didCompleteWithError (with an error that is not 'canceled') or in other scenarios where we want to handle the failure and then trigger a cancel (see the challenge handling).